yagooglesearch/__init__.py

```diff
@@ -461,6 +465,25 @@ def search(self):
                     ROOT_LOGGER.warning(f"No href for link: {link}")
                     continue
 
+                if (self.output == "complete"):
+                    # Get the first SPAN from the anchor tag.
+                    try:
+                        title = a.get_text()
+                    except Exception:
+                        ROOT_LOGGER.warning(f"No title and desc for link")
+                        title = ''
+                        continue
```

Comment:
With this `continue`, it will go to the top of the for loop and not populate a "complete" dictionary for the result found.  So setting `title=""` does nothing.

The logic is if there is no `title` (same for the `desc` found later in the code), that search result would be discarded.  Are you looking to have a result populated even if the `title` or `desc` isn't set found?  I'd say yes (option 1), but wanted to confirm.

1)  Option 1, despite not all values being populated, the dictionary will be **still** added to the `unique_complete_result` list.

```python
{
    "url": "https://github.com",
    "title": "GitHub site",
    "desc": "",
}
```


2) Option 2, not all values are populated, discard and don't return as part of the `unique_complete_result` list.

```python
{
    "url": "https://example.com",
    "title": "",
    "desc": "Example site description",
}
```

Reply:
I need it to behave like option 1.
You are right, both 'continue' interfere with the behavior of option 1. I would only discard if the link does not exist.